src/app/file-manager/services/login-api.service.ts

```diff
@@ -28,10 +29,10 @@ const googleOptions = {
 
 @Injectable()
 export class LoginApiService extends LoginService {
```

Comment:
This could use a better name; 'API' doesn't seem like a great way to distinguish this from `LoginService`.  How about `GoogleLoginService`?